lib/import/colorectal/providers/birmingham/variant_processor.rb

```diff
@@ -19,10 +18,12 @@ def initialize(genocolorectal, record, logger)
             @genelist       = COLORECTAL_GENES_MAP[@record.raw_fields['indication']]
           end
 
+          # rubocop:disable Metrics/AbcSize
```

Comment:
Please  leave rubocop warnings in place so that we cab look at fixing them, possibly at a later date